src/styles/style.css

```diff
@@ -0,0 +1,163 @@
+.body {
+  margin: 0;
+
+  --border-color: rgb(240, 7, 228);
+  --hover-color: blue;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 199px;
+  height: 100%;
+  border: 1px solid var(--border-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image {
+  box-sizing: border-box;
+  background-image: url(/src/images/imac.jpeg);
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  margin-top: 33px;
+  margin-left: 19px;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  padding: 40px 16px 4px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  padding: 4px 16px;
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+}
+
+.card:hover {
+  outline: #00acdc;
+  border: 1px solid var(--hover-color);
+  border-radius: 5px;
+  box-shadow: #ed0303;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 24px 16px 0;
+}
+
+.card__price-key {
+  width: 31px;
+  height: 18px;
+
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+  color: #616070;
+}
+
+.card__price-value {
+  width: 50px;
+  height: 18px;
+
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+  color: #060b35;
+}
+
+.review {
+  display: flex;
+  flex-direction: row;
+  padding: 16px 16px 0;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.review__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+}
+
+.star-active {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+  box-sizing: border-box;
+  align-items: center;
+  margin: 0 4px 0 0;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  box-sizing: border-box;
+  align-items: center;
+  margin: 0 4px 0 0;
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
The `.star-active` and `.star` classes share many common styles. Consider creating a base class for the shared properties to follow the DRY (Don't Repeat Yourself) principle.